### ISSUE TITLE
Memory test: phet brand

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#269]
see https://github.com/phetsims/ratio-and-proportion/issues/210

**Comment by zepumph** [comment]
After 30 minutes of testing, here are the snapshots:

![image](https://user-images.githubusercontent.com/6856943/101421191-40eb0900-38a0-11eb-8aea-c408edc62697.png)


The first 6 were every 1 minute, then the next 3 were every 2 minutes, then the last 2 were after 10 minutes. 

It was constantly increasing so perhaps there is  a memory leak, but a very minor one at that. 

Here is what the comparison will look like for comparing 11 to 9:
![image](https://user-images.githubusercontent.com/6856943/101421383-a808bd80-38a0-11eb-82a3-2616ca6db4b6.png)


Then I realized my tab was muted, so I unmuted it, and then fuzzed for a few more seconds and compared it to the previous, and these audio-related types were cleaned up:

![image](https://user-images.githubusercontent.com/6856943/101421520-eef6b300-38a0-11eb-980e-8240089a17a6.png)


I'm feeling good here! I'll leave open for the code reviewer.
